app/src/androidTest/java/com/hedvig/app/util/KakaoExtensions.kt

```diff
@@ -21,7 +24,16 @@ fun KTextInputLayout.hasError(@StringRes resId: Int, vararg formatArgs: Any) =
 fun KDatePicker.setDate(date: LocalDate) = setDate(date.year, date.monthValue, date.dayOfMonth)
 
 fun KTextView.hasText(@StringRes resId: Int, vararg formatArgs: Any) =
-    hasText(context().getString(resId, *formatArgs))
+    view.check { view, noViewFoundException ->
+        if (view is TextView) {
+            val text = view.resources.getString(resId, *formatArgs)
+            if (text != view.text) {
```

Comment:
awesome

Reply:
I'm not actually sure that it's required any more though given that I defaulted to using a translated context in the utility function 😬